docs/ENV.md

```diff
@@ -34,6 +44,27 @@ The database user.
 #### `CUBEJS_DB_PASS`
 The password for the database user.
 
+### Database settings (for BigQuery)
+You may wish to use BigQuery for your staging or production setup. However, if you want to connect from your local setup, you can do that as well. Configure the following env variables to connect your Cube.js instance to your BigQuery instance. These variables are not defined in `.env.example`.
+
+#### `CUBEJS_DB_TYPE`
+Database engine for Plio. It should be `bigquery`.
+
+#### `CUBEJS_DB_BQ_PROJECT_ID`
+The project id for your bigquery project. Generally, it's the name of your project.
+
+#### `CUBEJS_DB_BQ_CREDENTIALS`
+This is a base64 encoded value of your Google Cloud Platform's service account. You can learn more about acquiring service account credentials [here](https://cloud.google.com/docs/authentication/getting-started) and [here](https://console.cloud.google.com/projectselector2/iam-admin/serviceaccounts?supportedpurview=project). The service account must have BigQuery admin permissions.
+
+Once you have downloaded the JSON file, run the following commands and use the output for this environment variable:
+```sh
+cat /path/to/gcp-service-account-filename.json | base64
+```
+
+#### `CUBEJS_DB_BQ_LOCATION`
+The region of your bigquery setup in Google Cloud Console. You can check this from the dataset info section mentioned above. You can find all supported regions [here](https://cloud.google.com/bigquery/docs/locations#regional-locations).
```

Comment:
I don't see any dataset info section mentioned above - am I missing something?

Reply:
oh i think I had it added, then removed. Updating this statement.